dns_server/src/main.rs

```diff
@@ -58,8 +58,9 @@ async fn run(config: Arc<DnsServerConfig>) -> Result<Never, error::DnsServerErro
         boot_nodes: Vec::new(),
         reserved_nodes: Vec::new(),
         whitelisted_addresses: Default::default(),
-        ban_threshold: Default::default(),
-        ban_duration: Default::default(),
+        // Note: this ban config (as well as any other non-backend setting here) won't have
```

Comment:
Please define "non-backed". If it's something that isn't used in the DNS server, specifying them one by one helps.